controllers/videos.js

```diff
@@ -5,21 +5,21 @@ const auth = require('../middleware/auth');
 
 //Get all videos
 router.get('/', auth, (req, res) => {
-	db.Video.find({ user: req.user })
+	db.Video.find({ userId: req.user })
```

Comment:
you need to get  the userId from the request. 
const userId = req.userId; 
req.userId is the value you got from decode.id in auth.js.
the you can just call db.Video.find({userId});
you do not need to send the userId from the front-end.